src/Microsoft.AspNetCore.SignalR.Common/Internal/Protocol/MessagePackHubProtocol.cs

```diff
@@ -132,6 +136,16 @@ private static CompletionMessage CreateCompletionMessage(Unpacker unpacker, IInv
             return new CompletionMessage(invocationId, error, result, hasResult);
         }
 
+        private static StreamCompletionMessage CreateStreamCompletionMessage(Unpacker unpacker, long arraySize, IInvocationBinder binder)
+        {
+            Debug.Assert(arraySize == 2 || arraySize == 3, "Unexpected item count");
+
+            var invocationId = ReadInvocationId(unpacker);
+            // Error is optional so StreamCompletion without error will has 3 items, StreamCompletion with error has 4 items
```

Comment:
"will have" and it should be 2 and 3?

Reply:
should be 2 and 3 - 3 and 4 was a bug which I found by running end to end JS test. Fixed the code but missed the comment